unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/CharacterController/DCLCharacterController.cs

```diff
@@ -472,14 +473,18 @@ public Transform CastGroundCheckingRays()
         return null;
     }
 
-    public bool CastGroundCheckingRays(float extraDistance, float scale, out RaycastHit hitInfo) { return CastGroundCheckingRays(transform, collider, extraDistance, scale, groundLayers, out hitInfo); }
+    public bool CastGroundCheckingRays(float extraDistance, float scale, out RaycastHit hitInfo)
+    {
+        bool raycastSucceded = CastGroundCheckingRays(transform, collider, extraDistance, scale, groundLayers, out hitInfo);
+        return IsLastCollisionGround() || raycastSucceded;
```

Comment:
```suggestion
        if(CastGroundCheckingRays(transform, collider, extraDistance, scale, groundLayers, out hitInfo))
                return true;
        return IsLastCollisionGround();
```

More nitpicking, Isn't this equivalent and more readable?